Automattic/wp-calypso
# Remove pricing page SPP variant

### Changes proposed in this Pull Request

This PR removes the variant of the pricing page named SPP, that was not conclusive as part of the `jetpackSimplifyPricingPage` experiment.

### Implementation notes

- The `i5` variant is the only variant left for now
- I left the Jetpack CRM Free card in the repo since it's likely to be reused

### Testing instructions

- Download this PR
- Run both cloud and Calypso concurrently
- Open your browser console
- Check that the plans/pricing page looks and behaves as the `control` variant of the `jetpackSimplifyPricingPage` experiment in production
- Check that the `jetpackSimplifyPricingPage` experiment is not shown in the a/b test menu anymore
- Verify there's no error in the console
